app/routes.py

```diff
@@ -1,2 +1,160 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.models.helpers import validate_planet
 
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+# CREATE PLANET
+@planets_bp.route("", methods = ["POST"])
+def create_planet():
+	request_body = request.get_json()
+	
+	new_planet = Planet.create(request_body)
+
+	db.session.add(new_planet)
+	db.session.commit()
+
+	return jsonify(f"Planet {new_planet.name} has been successfully created"), 201 #use make response when you want to return something that is not json
+
+# GET ALL
+@planets_bp.route("", methods = ["GET"])
+def get_all_planets():
+
+	days_query = request.args.get("length_of_year")
+	name_query = request.args.get("name")
+	description_query = request.args.get("description")
+	circumference_query = request.args.get("circumference")
+
+	if days_query:
+		planets = Planet.query.filter_by(length_of_year = days_query)
+	elif name_query:
+		planets = Planet.query.filter_by(name = name_query)
+	elif description_query:
+		planets = Planet.query.filter_by(description = description_query)
+	elif circumference_query:
+		planets = Planet.query.filter_by(circumference = circumference_query)
+	else:
+		planets = Planet.query.all()
+
+	planets_response = []
+	
+	for planet in planets:
+		planets_response.append(planet.to_json())
+			
+	return jsonify(planets_response)
+
+# GET ONE PLANET
+@planets_bp.route("/<id>", methods = ["GET"])
+def get_one_planet(id):
+    planet = validate_planet(id)
+    return jsonify(planet.to_json()), 200
```

Comment:
Nice helper! Also, if a view function returns a dictionary (`planet.to_json()` in this case), Flask will automatically convert the dictionary into a JSON response object (aka Flask will turn dictionaries into JSON data for us). This means you can leave out `make_response()` like so:

```python
@planets_bp.route("/<planet_id>", methods=["GET"])
def read_one_planet(planet_id):
    planet = validate_planet(planet_id)
    return planet.to_json(), 200
```

Here's the documentation with more info:
https://flask.palletsprojects.com/en/2.0.x/quickstart/#apis-with-json